api/dsym.md

```diff
@@ -0,0 +1,23 @@
+## Upload your .dSYM files
+
+	POST /api/1/dsym
+
+Requires `post_server_item` scope.
```

Comment:
I think you can drop this since you cover it in the Form Data Parameters section